horovod/common/ops/nccl_operations.cc

```diff
@@ -148,10 +148,13 @@ void NCCLOpContext::PopulateNCCLCommStrategy(int& nccl_rank, int& nccl_size,
   } else if (communicator_type_ == Communicator::LOCAL) {
     nccl_rank = process_set.controller->GetLocalRank();
     nccl_size = process_set.controller->GetLocalSize();
+  } else if (communicator_type_ == Communicator::CROSS) {
+    nccl_rank = process_set.controller->GetCrossRank();
+    nccl_size = process_set.controller->GetCrossSize();
   } else {
-    throw std::logic_error("Communicator type " +
-                           std::to_string(communicator_type_) +
-                           " is not supported in NCCL mode.");
+     throw std::logic_error("Communicator type " +
```

Comment:
Nit: Extraneous spaces added here should be removed.

Reply:
Hi @romerojosh thanks for catching this! Updated now